Staging next

FRidh opened the pull request:
###### Motivation for this change


###### Things done

<!-- Please check what applies. Note that these are not hard requirements but merely serve as information for reviewers. -->

- [ ] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

jtojnar committed:
pan: clean up

jtojnar committed:
gnome3.gnome-keyring: clean up

jtojnar committed:
xfce.xfce4-settings: remove unneeded dependency

https://bluesabre.org/2018/06/20/xfce-settings-4-12-4-and-4-13-4-released/

jtojnar committed:
modemmanager: remove unneeded dependency

jtojnar committed:
umockdev: split package

jtojnar committed:
gnome3.gnome-desktop: use substituteAll for patch

FRidh committed:
python: multipledispatch: 0.5.0 -> 0.6.0

FRidh committed:
python: mutagen: 1.41.0 -> 1.41.1

FRidh committed:
python: netCDF4: 1.4.0 -> 1.4.1

FRidh committed:
python: nipype: 1.1.0 -> 1.1.2

FRidh committed:
python: numexpr: 2.6.6 -> 2.6.7

FRidh committed:
python: numpy: 1.14.5 -> 1.15.0

FRidh committed:
python: oauthenticator: 0.7.3 -> 0.8.0

FRidh committed:
python: openpyxl: 2.5.4 -> 2.5.5

FRidh committed:
python: pandas: 0.23.3 -> 0.23.4

FRidh committed:
python: pbr: 4.1.1 -> 4.2.0

FRidh committed:
python: pendulum: 2.0.2 -> 2.0.3

FRidh committed:
python: persistent: 4.2.4.2 -> 4.3.0

FRidh committed:
python: phonopy: 1.13.2.13 -> 1.13.2.107

FRidh committed:
python: plac: 0.9.6 -> 1.0.0

FRidh committed:
python: plotly: 3.1.0 -> 3.1.1

FRidh committed:
python: pluggy: 0.6.0 -> 0.7.1

FRidh committed:
python: plumbum: 1.6.6 -> 1.6.7

FRidh committed:
python: plyfile: 0.5 -> 0.6

FRidh committed:
python: prometheus_client: 0.3.0 -> 0.3.1

FRidh committed:
python: py3status: 3.11 -> 3.12

FRidh committed:
python: pyaxmlparser: 0.3.9 -> 0.3.10

FRidh committed:
python: pycryptodome: 3.6.4 -> 3.6.5

FRidh committed:
python: pycryptodomex: 3.6.4 -> 3.6.5

FRidh committed:
python: pycuda: 2017.1.1 -> 2018.1

FRidh committed:
python: pylast: 2.3.0 -> 2.4.0

FRidh committed:
python: pylint: 2.0.1 -> 2.1.1

FRidh committed:
python: PyLTI: 0.6.0 -> 0.7.0

FRidh committed:
python: pymatgen: 2018.6.27 -> 2018.8.10

FRidh committed:
python: pyowm: 2.8.0 -> 2.9.0

FRidh committed:
python: pytest: 3.6.3 -> 3.7.1

FRidh committed:
python: pytest-asyncio: 0.8.0 -> 0.9.0

FRidh committed:
python: pytest-flakes: 3.0.2 -> 4.0.0

FRidh committed:
python: pytest-xdist: 1.22.3 -> 1.22.5

FRidh committed:
python: python-ctags3: 1.2.4 -> 1.3.0

FRidh committed:
python: python_openzwave: 0.4.7 -> 0.4.9

FRidh committed:
python: pyunifi: 2.13 -> 2.14

FRidh committed:
python: pyzmq: 17.1.0 -> 17.1.2

FRidh committed:
python: rlp: 1.0.1 -> 1.0.2

FRidh committed:
python: rope: 0.10.7 -> 0.11.0

FRidh committed:
python: setuptools_scm: 2.1.0 -> 3.1.0

FRidh committed:
python: shippai: 0.2.3 -> 0.3.1

FRidh committed:
python: simpleeval: 0.9.5 -> 0.9.6

FRidh committed:
python: snakeviz: 0.4.1 -> 1.0.0

FRidh committed:
python: spglib: 1.10.4.1 -> 1.10.4.11

FRidh committed:
python: sqlmap: 1.2.7 -> 1.2.8

FRidh committed:
python: stripe: 2.0.3 -> 2.4.0

FRidh committed:
python: sybil: 1.0.8 -> 1.0.9

FRidh committed:
python: tox: 3.1.2 -> 3.2.1

FRidh committed:
python: tqdm: 4.23.4 -> 4.24.0

FRidh committed:
python: txaio: 2.10.0 -> 18.7.1

FRidh committed:
python: typeguard: 2.2.0 -> 2.2.1

FRidh committed:
python: uproot: 2.9.6 -> 2.9.11

FRidh committed:
python: voluptuous: 0.11.1 -> 0.11.5

FRidh committed:
python: whitenoise: 4.0b4 -> 4.0

FRidh committed:
python: widgetsnbextension: 3.3.0 -> 3.4.0

FRidh committed:
python: zeep: 3.0.0 -> 3.1.0

FRidh committed:
python: zodbpickle: 1.0.1 -> 1.0.2

FRidh committed:
pythonPackages.backports_lzma: fix build

FRidh committed:
pythonPackages.pytest_32: move into celery

as that's the only consumer of this version of Pytest.

FRidh committed:
pythonPackages.jinja2: disable tests

FRidh committed:
python: absl-py: 0.3.0 -> 0.4.0

FRidh committed:
python: aiohue: 1.6.0 -> 1.7.0

FRidh committed:
python: allpairspy: 2.4.0 -> 2.4.1

FRidh committed:
python: APScheduler: 3.5.2 -> 3.5.3

FRidh committed:
python: argon2_cffi: 18.1.0 -> 18.3.0

FRidh committed:
python: autobahn: 18.7.1 -> 18.8.1

FRidh committed:
python: bitmath: 1.3.1.2 -> 1.3.3.1

FRidh committed:
python: bleach: 2.1.3 -> 2.1.4

FRidh committed:
python: botocore: 1.10.75 -> 1.11.1

FRidh committed:
python: bugwarrior: 1.5.1 -> 1.6.0

FRidh committed:
python: cement: 2.10.12 -> 3.0.0

FRidh committed:
python: certifi: 2018.8.13 -> 2018.8.24

FRidh committed:
python: cftime: 1.0.0 -> 1.0.1

FRidh committed:
python: chainer: 4.3.1 -> 4.4.0

FRidh committed:
python: channels: 2.1.2 -> 2.1.3

FRidh committed:
python: CherryPy: 17.0.0 -> 17.3.0

FRidh committed:
python: cloudpickle: 0.5.3 -> 0.5.5

FRidh committed:
python: cmd2: 0.9.3 -> 0.9.4

FRidh committed:
python: coveralls: 1.3.0 -> 1.4.0

FRidh committed:
python: cryptography: 2.3 -> 2.3.1

FRidh committed:
python: cryptography_vectors: 2.3 -> 2.3.1

FRidh committed:
python: cupy: 4.3.0 -> 4.4.0

FRidh committed:
python: python-debian: 0.1.32 -> 0.1.33

FRidh committed:
python: detox: 0.12 -> 0.13

FRidh committed:
python: easydict: 1.7 -> 1.8

FRidh committed:
python: fastrlock: 0.3 -> 0.4

FRidh committed:
python: filelock: 3.0.4 -> 3.0.6

FRidh committed:
python: gevent: 1.3.5 -> 1.3.6

FRidh committed:
python: IBMQuantumExperience: 2.0.1 -> 2.0.2

FRidh committed:
python: jsonrpc-async: 1.0.0 -> 1.0.1

FRidh committed:
python: jsonrpc-base: 1.0.1 -> 1.0.2

FRidh committed:
python: jsonrpc-websocket: 1.0.0 -> 1.0.1

FRidh committed:
python: jupyterlab: 0.33.10 -> 0.34.2

FRidh committed:
python: Keras_Applications: 1.0.4 -> 1.0.5

FRidh committed:
python: Keras_Preprocessing: 1.0.2 -> 1.0.3

FRidh committed:
python: kubernetes: 6.0.0 -> 7.0.0

FRidh committed:
python: libusb1: 1.6.5 -> 1.6.6

FRidh committed:
python: mozdevice: 1.0.0 -> 1.0.1

FRidh committed:
python: marshmallow-sqlalchemy: 0.14.0 -> 0.14.1

FRidh committed:
python: ncclient: 0.6.0 -> 0.6.2

FRidh committed:
python: numpy: 1.15.0 -> 1.15.1

FRidh committed:
python: pathspec: 0.5.6 -> 0.5.7

FRidh committed:
python: persistent: 4.3.0 -> 4.4.1

FRidh committed:
python: phonenumbers: 8.9.10 -> 8.9.11

FRidh committed:
python: pkgconfig: 1.3.1 -> 1.4.0

FRidh committed:
python: psutil: 5.4.6 -> 5.4.7

FRidh committed:
python: pycryptodome: 3.6.5 -> 3.6.6

FRidh committed:
python: pycryptodomex: 3.6.5 -> 3.6.6

FRidh committed:
python: pyhomematic: 0.1.46 -> 0.1.47

FRidh committed:
python: pyodbc: 4.0.23 -> 4.0.24

FRidh committed:
python: Pyphen: 0.9.4 -> 0.9.5

FRidh committed:
python: pytest: 3.7.1 -> 3.7.2

FRidh committed:
python: pytest-xdist: 1.22.5 -> 1.23.0

FRidh committed:
python: python-daemon: 2.1.2 -> 2.2.0

FRidh committed:
python: python-docx: 0.8.6 -> 0.8.7

FRidh committed:
python: qtconsole: 4.3.1 -> 4.4.1

FRidh committed:
python: regex: 2018.07.11 -> 2018.08.17

FRidh committed:
python: reikna: 0.7.0 -> 0.7.1

FRidh committed:
python: reportlab: 3.5.2 -> 3.5.6

FRidh committed:
python: s3fs: 0.1.5 -> 0.1.6

FRidh committed:
python: setuptools: 40.0.0 -> 40.2.0

FRidh committed:
python: soco: 0.15 -> 0.16

FRidh committed:
python: Sphinx: 1.7.6 -> 1.7.7

FRidh committed:
python: SQLAlchemy: 1.2.10 -> 1.2.11

FRidh committed:
python: stripe: 2.4.0 -> 2.6.0

FRidh committed:
python: svg.path: 2.2 -> 3.0

FRidh committed:
python: tqdm: 4.24.0 -> 4.25.0

FRidh committed:
python: trio: 0.5.0 -> 0.6.0

FRidh committed:
python: typeguard: 2.2.1 -> 2.2.2

FRidh committed:
python: vega: 1.3.0 -> 1.4.0

FRidh committed:
python: websocket_client: 0.48.0 -> 0.51.0

FRidh committed:
python: XlsxWriter: 1.0.5 -> 1.0.7

FRidh committed:
pythonPackages.pytest-asyncio: fix build

FRidh committed:
pythonPackages.bootstrapped-pip: setuptools: 40.0.0 -> 40.2.0

FRidh committed:
mypy: use toPythonApplication

Top-level Python packages are applications, not libraries.

Commit:
[cpan2nix] perlPackages.ArchiveZip: 1.60 -> 1.62

Commit:
[cpan2nix] perlPackages.CPANPerlReleases: 3.70 -> 3.74

Commit:
[cpan2nix] perlPackages.CpanelJSONXS: 4.04 -> 4.05

Commit:
[cpan2nix] perlPackages.Filter: 1.58 -> 1.59

Commit:
[cpan2nix] perlPackages.LogAny: 1.706 -> 1.707

Commit:
[cpan2nix] perlPackages.MLDBM: cleanup

Commit:
[cpan2nix] perlPackages.ModuleCoreList: 5.20180720 -> 5.20180820

Commit:
[cpan2nix] perlPackages.Mojolicious: 7.88 -> 7.93

Commit:
[cpan2nix] perlPackages.NetPing: 2.68 -> 2.70

Commit:
[cpan2nix] perlPackages.Paranoid: 2.05 -> 2.06

Commit:
[cpan2nix] perlPackages.PathTiny: 0.106 -> 0.108

Commit:
[cpan2nix] perlPackages.PkgConfig: 0.19026 -> 0.21026

Commit:
[cpan2nix] perlPackages.TestSimple13: 1.302138 -> 1.302140

Commit:
[cpan2nix] perlPackages.TextCSV: 1.95 -> 1.97

Commit:
[cpan2nix] perlPackages.IOSocketSSL: 2.058 -> 2.059

Commit:
[cpan2nix] perlPackages.ModuleScanDeps: 1.24 -> 1.25

Commit:
[cpan2nix] perlPackages.TypeTiny: 1.002002 -> 1.004002

Commit:
[cpan2nix] perlPackages.XMLParserLite: 0.721 -> 0.722

Commit:
[cpan2nix] perlPackages.NetDNS: 1.16 -> 1.17

Commit:
[cpan2nix] perlPackages.TextSimpleTable: 2.05 -> 2.07

Commit:
[cpan2nix] perlPackages.TextQuoted: 2.09 -> 2.10

Commit:
[cpan2nix] perlPackages.CGISimple: 1.15 -> 1.16

Commit:
[cpan2nix] perlPackages.CarpAlways: 0.13 -> 0.16

Commit:
[cpan2nix] perlPackages.Coro: 6.514 -> 6.52

Commit:
[cpan2nix] perlPackages.PodMarkdown: 3.005 -> 3.101

volth committed:
[cpan2nix] perlPackages.CGI: 4.38 -> 4.40

volth committed:
[cpan2nix] perlPackages.FileMimeInfo: 0.28 -> 0.29

volth committed:
[cpan2nix] perlPackages.MonitoringPlugin: 0.39 -> 0.40

volth committed:
[cpan2nix] perlPackages.GamesSolitaireVerify: 0.1800 -> 0.1900

volth committed:
[cpan2nix] perlPackages.GetoptLongDescriptive: 0.102 -> 0.103

volth committed:
[cpan2nix] perlPackages.ProtocolHTTP2: 1.08 -> 1.09

volth committed:
[cpan2nix] perlPackages.PPIxRegexp: 0.061 -> 0.062

volth committed:
[cpan2nix] perlPackages.XMLSemanticDiff: 1.0006 -> 1.0007

volth committed:
[cpan2nix] perlPackages.Mouse: v2.5.4 -> v2.5.6

volth committed:
[cpan2nix] perlPackages.ApacheAuthCookie: cleanup

volth committed:
[cpan2nix] perlPackages.RESTClient: cleanup

volth committed:
[cpan2nix] perlPackages.ParamsValidationCompiler: 0.27 -> 0.30

volth committed:
[cpan2nix] perlPackages.NetDBus: cleanup

volth committed:
[cpan2nix] perlPackages.DateTime: 1.49 -> 1.50

volth committed:
[cpan2nix] perlPackages.MooseXGetopt: 0.71 -> 0.72

volth committed:
[cpan2nix] perlPackages.DBFile: 1.841 -> 1.842

FRidh committed:
Merge pull request #45572 from r-ryantm/auto-update/hdf5-threadsafe

hdf5-threadsafe: 1.10.2 -> 1.10.3

FRidh committed:
awscli: 1.15.66 -> 1.16.1

gebner committed:
mesa: enable texture floats

FRidh committed:
python: numexpr: 2.6.7 -> 2.6.8

FRidh committed:
Merge pull request #44973 from NixOS/python-unstable

Python packages set: last major updates before branch off

FRidh committed:
Merge pull request #44752 from costrouc/costrouc/lammps-update

material science packages (init and update): lammps, python - ase, pymatgen-lammps, lammps-cython, dftfit

gebner committed:
Merge pull request #45631 from gebner/mesa-texture-floats

mesa: enable texture floats

jtojnar committed:
Merge pull request #45627 from jtojnar/gnome-cleanup

Gnome cleanup

xeji committed:
Merge pull request #45629 from volth/cpan2nix-2018-08-25

perlPackages: cpan2nix updates

vcunat committed:
mesa: 18.1.5 -> 18.1.7 (maintenance)

FRidh committed:
Merge master into staging

volth committed:
perlPackages.AlienWxWidgets: fix build

volth committed:
perlPackages.ArchiveZip: fix build

timokau committed:
Merge pull request #45257 from timokau/openblas-pc

openblas: add pkg-config aliases

timokau committed:
Merge pull request #45654 from volth/patch-237

perlPackages.ArchiveZip: fix staging build

timokau committed:
Merge pull request #45653 from volth/patch-236

perlPackages.AlienWxWidgets: fix staging build

volth committed:
perlPackages.SVNSimple: fix build with non-default perl

Mic92 committed:
Merge pull request #45661 from volth/patch-238

perlPackages.SVNSimple: fix build with non-default perl

r-ryantm committed:
e2fsprogs: 1.44.3 -> 1.44.4 (#45602)

Semi-automatic update generated by
https://github.com/ryantm/nixpkgs-update tools. This update was made
based on information from
https://repology.org/metapackage/e2fsprogs/versions.

FRidh committed:
Merge master into staging

FRidh committed:
Merge staging into staging-next

FRidh committed:
Merge master into staging-next

vcunat committed:
gnome3.gnome-shell: fix build (missing xsltproc)

vcunat commented:
On the whole I see relatively lots of python package regressions https://hydra.nixos.org/eval/1476054?compare=1476050 so it would be nice if you had a look at reducing that.

FRidh committed:
python: absl-py: 0.4.0 -> 0.4.1

FRidh committed:
python: aws-adfs: 0.12.0 -> 0.12.1

FRidh committed:
python: botocore: 1.11.1 -> 1.11.4

FRidh committed:
python: deform: 2.0.5 -> 2.0.6

FRidh committed:
python: dftfit: 0.4.11 -> 0.4.22

FRidh committed:
python: Faker: 0.8.17 -> 0.8.18

FRidh committed:
python: joblib: 0.12.2 -> 0.12.3

FRidh committed:
python: jupyterlab: 0.34.2 -> 0.34.6

FRidh committed:
python: moto: 1.3.4 -> 1.3.5

FRidh committed:
python: openpyxl: 2.5.5 -> 2.5.6

FRidh committed:
python: persistent: 4.4.1 -> 4.4.2

FRidh committed:
python: phonenumbers: 8.9.11 -> 8.9.12

FRidh committed:
python: pytest: 3.7.2 -> 3.7.4

FRidh committed:
python: pytest-timeout: 1.3.1 -> 1.3.2

FRidh committed:
python: regex: 2018.08.17 -> 2018.08.29

FRidh committed:
python: sockjs-tornado: 1.0.3 -> 1.0.5

FRidh committed:
python: Sphinx: 1.7.7 -> 1.7.8

FRidh committed:
python: typing: 3.6.4 -> 3.6.6

FRidh committed:
python: XlsxWriter: 1.0.7 -> 1.0.9

FRidh committed:
pythonPackages.boto3: 1.7.57 -> 1.7.84

FRidh committed:
pythonPackages.folium: add missing numpy runtime dependency

FRidh committed:
pythonPackages.pytestrunner: failed attempt at enabling tests

FRidh commented:
@vcunat I'll push some to https://github.com/NixOS/nixpkgs/pull/45792 and see what happens.

FRidh committed:
pythonPackages.boto3: 1.7.84 -> 1.8.4

FRidh committed:
awscli: 1.16.1 -> 1.16.4

FRidh committed:
pythonPackages.cement: disable Python 2

Commit:
python.pkgs.sarge: disable tests

FRidh committed:
Revert "python: shippai: 0.2.3 -> 0.3.1"

This broke vdirsyncer.

This reverts commit 0bca28c89078ddc25aca0c1f8c50d3bafdd36412.

FRidh committed:
Revert "python.pkgs.python-jose: 2.0.2 -> 3.0.0"

This breaks moto.

This reverts commit a6d3577d3d9a5aee20a88eb92de8dee266354269.

FRidh committed:
pythonPackages.scipy: disable tests

We should try to get these enabled again asap.
https://github.com/scipy/scipy/pull/8871#issuecomment-417391433

FRidh committed:
Python: fix botocore/boto3/awscli mess

Revert "pythonPackages.boto3: 1.7.84 -> 1.8.4"

This reverts commit 06b37899300e0800e7a0ea632fcb5bf0f291e003.

Revert "python: botocore: 1.11.1 -> 1.11.4"

This reverts commit 3b0242d4c014a440b8c1c58cae812dc84496cc95.

Revert "python: botocore: 1.10.75 -> 1.11.1"

This reverts commit 996fd7996713cdedc7e128d9930432c79d637f69.

Revert "awscli: 1.16.1 -> 1.16.4"

This reverts commit 4880df9078bf024fd5a98afa8c6f28833e13f3d3.

Revert "awscli: 1.15.66 -> 1.16.1"

This reverts commit 6923bc8d23e2735708492da627a378de5a53c218.

awscli: fix build

pythonPackages.botocore: 1.10.75 -> 1.10.84

FRidh committed:
pythonPackages.moto: fix build

FRidh committed:
pythonPackages.cmd2: fix build

Commit:
awsebcli: use python3

Its dependency cement is disabled on python2 since 36a4df6.

FRidh committed:
pythonPackages.bottleneck: get rid of warnings during tests

Commit:
python.pkgs.keyrings-alt: fix tests

Commit:
python.pkgs.celery: fix tests

Commit:
python.pkgs.cairocffi: remove superfluous patch

Commit:
Merge branch 'master' into staging-next

Commit:
octoprint: pin pylru to 1.0.9

Commit:
python.pkgs.natsort: no pyflakes and pep8 checks

Commit:
python.pkgs.h5py: disable tests

Commit:
Merge branch 'master' into staging-next

FRidh committed:
pythonPackages.pandas: fix build

FRidh committed:
Merge master into staging-next

vcunat committed:
pythonPackages.more-itertools: avoid doCheck on i686

I expect noone will want to spend time really fixing these,
but it blocks wine from building...
